docs/quickstart_guide.md

```diff
@@ -207,6 +245,18 @@ Execute the following below **as a non-root user with sudo rights** using **bash
     ```bash
     ./scripts/cloudlab/setup_node.sh;
     ```
+    > **Note:**
+    > To enable runs with `stargz` images, setup kubelet by adding the `stock-only` and `use-stargz`
+    > flags as follows:
+    >   ```bash
+    >   ./scripts/cloudlab/setup_node.sh stock-only use-stargz
+    >   ```
+    > **IMPORTANT**
+    > Currently `stargz` is only supported in native kubelet contexts without firecracker. 
+    > Therefore, the following steps from this guide must **not** be executed:
+    > * `2.3 - Start firecracker-containerd in a background terminal named firecracker`,
```

Comment:
Some typos in steps not to execute; should be 1.3, 1.4, 1.5

Reply:
Updated.